spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -164,10 +168,17 @@ public String toString() {
     }
 
     public String listToString(List<List<Object>> rows) {
-        return rows.stream().map(row ->
-                row.stream().map(field -> field == null ? NULL_VALUE : field.toString())
-                        .collect(Collectors.joining(FIELD_DELIMITER))
-        ).collect(Collectors.joining(LINE_DELIMITER));
+        if (addDoubleQuotes && trimDoubleQuotes) {
```

Comment:
It looks like this feature only works if both `add_double_quotes` and `trim_double_quotes` are set to `true`. So is it necessary to enforce setting `trim_double_quotes` to true when `add_double_quotes` is `true`?